engine/server/library/serverGame/src/shared/ai/HateList.cpp

```diff
@@ -535,8 +535,8 @@ void HateList::addServerNpAutoDeltaVariables(Archive::AutoDeltaByteStream & stre
 // ----------------------------------------------------------------------
 int HateList::getTimeSinceLastUpdate() const
 {
-	time_t const currentTime = Os::getRealSystemTime();
-	time_t const timeSinceActivitiy = (currentTime - m_lastUpdateTime.get());
+	uint32_t const currentTime = Os::getRealSystemTime();
+	uint32_t const timeSinceActivitiy = (currentTime - m_lastUpdateTime.get());
 
```

Comment:
so here we're replacing time_t in which case we still want to be using uint64_t but `Os::getRealSystemTime()` is still a time_t as it wasn't adjusted in this changeset